cmd/plasmacli/subcmd/tx/broadcast.go

```diff
@@ -0,0 +1,106 @@
+package tx
+
+import (
+	"fmt"
+	"github.com/FourthState/plasma-mvp-sidechain/msgs"
+	"github.com/FourthState/plasma-mvp-sidechain/plasma"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"strings"
+)
+
+func BroadcastSigsCmd() *cobra.Command {
+	includeCmd.Flags().Bool(asyncF, false, "wait for transaction commitment synchronously")
+	return includeCmd
+}
+
+var broadcastSigsCmd = &cobra.Command{
```

Comment:
Although the anteHandler for this msg will fail if the sig is wrong, we should do client side validation here before sending off the message.